snap/hooks/post-refresh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+set -eu
+
+echo "post-refresh hook" >> /tmp/snap.log # TODO: Only for debugging
+
+PREVIOUS_VERSION=$(snapctl get previous-version)
+echo "Previous version: $PREVIOUS_VERSION" >> /tmp/snap.log
```

Comment:
Good idea, changed the file name to `snap-post-refresh` and changed it so that some of the logs are written to the journal (`journalctl -r -t authd-oidc` to get them)